k8s/api-claim0-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    io.kompose.service: api-claim0
+  name: api-claim0
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
ReadOnly ? the api process does not need write access to the volume.

Reply:
Changed